templates/citizen_project/_form.html.twig

```diff
@@ -13,37 +13,53 @@
 
 <div class="form__new" id="field-name">
     {{ form_label(form.name, 'Donnez un nom à votre projet :', {label_attr: {class: 'label__header--large'}}) }}
-    {{ form_errors(form.name) }}
-    {{ form_widget(form.name, {
-        attr: {
-            'class': 'form--full',
-            'placeholder': 'Un nom court et clair',
-        }
-    }) }}
-    <small>Ex: "Un stage pour tous"</small>
+    {% if form.name.vars.disabled %}
+        <div id="citizen_project_name_text">{{ form.vars.data.name }}</div>
+        <div class="visually-hidden">{{ form_widget(form.name) }}</div>
```

Comment:
do we still need to render the form element here?

Reply:
Yes, if no, they will be rendered with `form_rest()` of parent templates.